strict.js

```diff
@@ -0,0 +1,25 @@
+
+ function  strictEqual() {
+     let plyrsreq = 52
+     if (FBteams == plyrsreq) {
+         return 'You have the amount of players required, lets play!'
+         
+     } else {
+         return 'sorry, you do not have enough players keep recruiting, try again next year!'
```

Comment:
All you need to do here is simply build a function no if else statement just a single function that  will find strict equality between two variables.